# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/tps/tps/testrunner.py
# Commit: b05343a5b533
# Full Hash: b05343a5b5333cc4eb4a15deac29eb3f5e0d1eec
# Author: Sammy Khamis <skhamis@mozilla.com>
# Date: 2023-03-01 03:56:13
# Regressor Bug: 1818349
# File Overlap Count: 1
# Overlapping Files: testing/tps/tps/testrunner.py
# Description:
#   Bug 1818349 Part 2: Update tabs TPS tests to allow using data: schema r=markh
#   
#   Depends on D171181
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D171182
# ==============================================================================

diff -r 57b2cda7e0db -r b05343a5b533 testing/tps/tps/testrunner.py
--- a/testing/tps/tps/testrunner.py	Tue Feb 28 05:14:29 2023 +0000
+++ b/testing/tps/tps/testrunner.py	Tue Feb 28 05:14:29 2023 +0000
@@ -79,6 +79,8 @@
         # hrm - not sure what the release/beta channels will do?
         "xpinstall.signatures.required": False,
         "services.sync.testing.tps": True,
+        # removed data: schema restriction for easier testing of tabs
+        "services.sync.engine.tabs.filteredSchemes": "about|resource|chrome|file|blob|moz-extension",
         "engine.bookmarks.repair.enabled": False,
         "extensions.experiments.enabled": True,
         "webextensions.storage.sync.kinto": False,
